x-pack/plugins/security_solution/public/explore/components/risk_score/risk_details_tab_body/index.tsx

```diff
@@ -86,12 +85,12 @@ const RiskDetailsTabBodyComponent: React.FC<
     () => (entityName ? buildEntityNameFilter([entityName], riskEntity) : {}),
     [entityName, riskEntity]
   );
-  const isChartEmbeddablesEnabled = useIsExperimentalFeatureEnabled('chartEmbeddablesEnabled');
+
   const { data, loading, refetch, inspect, isDeprecated, isModuleEnabled } = useRiskScore({
     filterQuery,
     onlyLatest: false,
     riskEntity,
-    skip: (!overTimeToggleStatus && !contributorsToggleStatus) || isChartEmbeddablesEnabled,
```

Comment:
The `Top contributor table` is still relying on this result when feature flag is on, so I shouldn't have skipped it here